tests/python/relay/test_backend_graph_executor.py

```diff
@@ -322,5 +323,17 @@ def test_graph_executor_api():
     assert mod.get_input_index("Invalid") == -1
 
 
+@tvm.testing.requires_llvm
+def test_benchmark():
+    mod, params = mlp.get_workload(1)
+    lib = relay.build(mod, target="llvm", params=params)
+    exe = graph_executor.create(lib.get_graph_json(), lib.lib, tvm.cpu())
+    data = tvm.nd.array(np.random.rand(1, 1, 28, 28).astype("float32"))
+    result = exe.benchmark(tvm.cpu(), data=data, func_name="run", repeat=2, number=1)
```

Comment:
A better test here would be to use unittest.patch() on the time_evaluator function itself so it always returns deterministic times, and hence you can fully test the behavior of the benchmark() function by itself independent of the actual module or time_evaluator behavior.